docs/src/components/ComponentDoc/ComponentPropsTable/ComponentPropsRow.tsx

```diff
@@ -1,68 +1,76 @@
-import * as PropTypes from 'prop-types'
 import * as React from 'react'
 import * as _ from 'lodash'
+import { Link } from 'react-router-dom'
 
-import ComponentPropDefaultValue from '../ComponentProp/ComponentPropDefaultValue'
-import ComponentPropDescription from '../ComponentProp/ComponentPropDescription'
+import { ComponentProp, ComponentPropType } from 'docs/src/types'
 import ComponentPropName from '../ComponentProp/ComponentPropName'
 
-export default class ComponentPropsRow extends React.Component<any, any> {
-  static propTypes = {
-    defaultValue: PropTypes.string,
-    description: PropTypes.arrayOf(PropTypes.string),
-    name: PropTypes.string,
-    required: PropTypes.bool,
-    tags: PropTypes.array,
-    type: PropTypes.string,
-    value: PropTypes.oneOfType([PropTypes.string, PropTypes.arrayOf(PropTypes.string)]),
-  }
+const InlineMarkdown = React.lazy(() => import('../InlineMarkdown'))
+
+type ComponentPropsRowProps = ComponentProp
 
-  state: any = {}
+const ComponentPropValue: React.FunctionComponent<ComponentPropType> = props => {
+  const { name, parameters } = props
 
-  toggleEnums = () => this.setState({ showEnums: !this.state.showEnums })
+  if (!name) return <span>enum</span>
```

Comment:
what's happening here? :)

Reply:
Improved this part in https://github.com/stardust-ui/react/pull/1634/commits/0bd34495c8acfb26c47b43865fa2dd4f97e96134 👍